bin/rumoji

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env ruby
 # -*- encoding: utf-8 -*-
 
+$: << File.expand_path(File.join(File.dirname(__FILE__), '..', 'lib'))
+
+require 'rubygems'
```

Comment:
These two lines have to go. This bin is designed to be run through `bundle exec` and that should set up the load path properly.